web/job_runs_controller_test.go

```diff
@@ -20,8 +20,16 @@ type JobRun struct {
 	ID string `json:"id"`
 }
 
+func BenchmarkJobRunsController_Index(b *testing.B) {
+	testJobRunsControllerIndex(b)
```

Comment:
I believe you need to [run the benchmarks in a loop](https://dave.cheney.net/2013/06/30/how-to-write-benchmarks-in-go), like:
```
for n := 0; n < b.N; n++ {
	testJobSpecsControllerIndex(b)
}
```

Reply:
I thought this Steve, but when you run the benchmarks it definitely runs them numerous times. I was seeing them run around 6-7 times to get the average like you were saying.